src/bridging/BridgingSdk/getQuoteWithBridging.ts

```diff
@@ -224,10 +266,7 @@ async function getBridgeResult(params: {
   // Get the pre-authorized hook
   const bridgeHook = await provider.getSignedHook(bridgeRequest.sellTokenChainId, unsignedBridgeCall, signer)
 
-  // Generate the app data for the hook
-  const metadataApi = new MetadataApi()
-  const appData = await metadataApi.generateAppDataDoc({
-    ...swapResult.appDataInfo.doc,
+  const appDataInfo = await mergeAppDataDoc(swapResult.appDataInfo.doc, {
```

Comment:
Probably it's not enough, because in CoW Swap I see the appData is not what I expceted

Reply:
Fixed: https://github.com/cowprotocol/cow-sdk/pull/275/commits/366882df04095534730cfb61e7067bf24c08c4bd